addons/im_livechat/views/rating_views.xml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <record id="rating_rating_view_form_livechat" model="ir.ui.view">
+        <field name="name">rating.rating.form.livechat</field>
+        <field name="model">rating.rating</field>
+        <field name="inherit_id" ref="rating.rating_rating_view_form"/>
+        <field name="mode">primary</field>
```

Comment:
Specific views should have a sequence to avoid having used as default views. Indeed sequence is used to choose the view to use when not having a view specified (all generic actions).